src/style.css

```diff
@@ -0,0 +1,108 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-grow: 1;
```

Comment:
Flex-grow is individual rule, don't add it to container.

Reply:
Ok, removed.